tasks/fh-build.js

```diff
@@ -39,6 +40,23 @@ module.exports = function(grunt) {
   }
 
   /**
+   * Keep just the first occurence of `node_modules` and the directory name next to it, and append `**`
+   * e.g. /home/user/work/fh-ngui/node_modules/watchify/node_modules/assert will be formatted  to node_modules/watchify/**
+   */
+  var formatDirectory = function(dir) {
+    var dirParts = dir.split(path.sep);
+    var nodeModules = dirParts.indexOf('node_modules');
+    return path.join(dirParts[nodeModules], dirParts[nodeModules + 1], '**');
+  };
+
+  /**
+   * Check if `node_modules` exists in a directory path
+   */
+  var nodeModulesExists = function(dir) {
+    return dir.indexOf('node_modules') >= 0
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png) Add a semicolon at the end of this statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ASemicolon)